src/scorer.ts

```diff
@@ -44,13 +55,91 @@ export class Scorer {
       inst.card.calcC?.(inst, this.allPlayerCards);
     });
 
-    const result: number[] = this.allPlayerCards.map((playerCards) => {
-      return playerCards.reduce(
-        (sum, inst) => sum + inst.finalA + inst.finalB || 0,
-        0
-      );
+    const playerScores: PlayerScore[] = this.allPlayerCards.map((playerCards) => {
+      // TODO AF: Add calcC when implemented
+      const playerCardsScores: CardScore[] = playerCards.map(c => {
+        const finalA = c.finalA;
+        const finalB = c.finalB ?? 0
+        const total = finalA + finalB;
+        return {finalA, finalB, total}
+      });
+
+      return new PlayerScore(playerCardsScores)
     });
 
-    return result;
+    const winningPlayersIndices = playerScores.reduce((maxScorePlayerIndices: number[], playerScore, index, arr) => {
+      const currentMax = arr[maxScorePlayerIndices[0]] // Get value of first max, but we can have multiple "max" values
+      if (playerScore.total === currentMax.total ) {
+        return [...maxScorePlayerIndices, index]
+      } else if (playerScore.total > currentMax.total) {
+        return [index]
```

Comment:
shouldn't this also update the `currentMax` value??

Reply:
No. The currentMax is calculated for each element in the array - no need to track the max.

Reply:
this has confussed me... but I will believe you that it works...